app/routes/surveys/components/SurveyEditor/Question.js

```diff
@@ -31,11 +31,11 @@ const questionTypeToIcon = {
 const QuestionTypeOption = (props: Object) => (
   <div
     className={cx(props.className, styles.dropdown)}
-    onMouseDown={() => {
+    onMouseDown={event => {
       props.onSelect && props.onSelect(props.option, event);
     }}
-    onMouseEnter={() => props.onFocus && props.onFocus(props.option, event)}
-    onMouseMove={() => {
+    onMouseEnter={event => props.onFocus && props.onFocus(props.option, event)}
```

Comment:
I remember having a discussion with someone about why I wasn't taking in `event` here, and how the code shouldn't work without it as I use the variable later. Did you change it just because it "should" be there, or did you run into any symptoms? Seems to work fine when I check out to this branch locally, I just want to be sure.

Reply:
The updated flow version told me to do it. And it makes no sense to don't have it. If it doesn't send anything, it will just behave the same way as before, making event undefined.